spec/features/merchants/dashboard_spec.rb

```diff
@@ -103,5 +103,14 @@
       expect(page).to have_content(@invoice_4.created_at.strftime("%A, %B %d,%Y"))
       expect(page).to have_content(@invoice_5.created_at.strftime("%A, %B %d,%Y"))
     end
+    it '#US36 lists items oldest to newest' do
+
+      within "div.items_ready_to_ship" do
```

Comment:
never seen this syntax before with div. - good to know

Reply:
Yea that's interesting-- so is it instead of using the` #`?